docs/application/dotnet/guides/nui/flexcontainer.md

```diff
@@ -325,31 +336,40 @@ You can modify the flex item placement through the following properties:
 
     To create a layout with the proportions shown in the above figure:
 
-    ```
+    ```cs
     /// Create the flex container
     FlexContainer flexContainer = new FlexContainer();
 
-    /// Place items horizontally
-    flexContainer.FlexDirection = FlexContainer.FlexDirectionType.Row;
+    /// Resize flex container to window size
+    flexContainer.WidthResizePolicy = ResizePolicyType.FillToParent;
+    flexContainer.HeightResizePolicy = ResizePolicyType.FillToParent;
+
+    /// Add the container to the window
+    Window.Instance.GetDefaultLayer().Add(flexContainer);
 
     /// Create flex items and set their proportions
     View item1 = new View();
+    item1.BackgroundColor = Color.Red;
     item1.Flex = 1.0f;
     flexContainer.Add(item1);
 
     View item2 = new View();
+    item2.BackgroundColor = Color.Green;
     item2.Flex = 3.0f;
     flexContainer.Add(item2);
 
     View item3 = new View();
+    item3.BackgroundColor = Color.Blue;
     item3.Flex = 1.0f;
     flexContainer.Add(item3);
 
     View item4 = new View();
+    item4.BackgroundColor = Color.Yellow;
     item4.Flex = 2.0f;
     flexContainer.Add(item4);
 
     View item5 = new View();
+    item5.BackgroundColor = Color.White;
```

Comment:
With this update, the background color of the items varies. However in the sample images, the background color is kept. I have a question about this. Do you have any plan to update the images also to match to the sample codes?

Reply:
I have treated it as a general view. But if we want to show directly this example, I can insert here screenshot from this example.